src/main/java/net/mcreator/generator/Generator.java

```diff
@@ -633,6 +634,7 @@ else if (listData instanceof Iterable<?> iterable) // fallback for the worst cas
 							conditionChecks.add(i, !TemplateExpressionParser.shouldSkipTemplateBasedOnCondition(this, conditionRaw, elements.get(i), operator));
 						}
 
+						// Only add GeneratorTemplatesList if at least one list template will be generated
 						if (!conditionChecks.contains(true) && performFSTasks)

```

Comment:
The comment does not mention usage of `performFSTasks` here - we still add GTL to the returned list if it's set to <kbd>false</kbd> (this way preview generator adds tabs for these GTLs upon its very initialization grouping them at the beginning of tabs list, otherwise it could append those tabs after normal files which is not OKay).
What should it say in this case? I want to change it to:
`// only add templates list if at least one list template will be generated or if we are in the preview mode`

Reply:
Generator system has no such thing as preview mode so this wording is not ok

Reply:
```
// Only add GeneratorTemplatesList if at least one list template will be generated and performFSTasks is true
// This is to make ... (your explanation here)
```

Reply:
```
// only add templates list if at least one list template will be generated
// or if we need all template lists possible for given mod element
```

Reply:
Goodish?

Reply:
```
// only add templates list if at least one list template will be generated
// or if we need all template lists possible for the given mod element (performFSTasks is false)
```